source/documentation/reference/operational-processes.md

```diff
@@ -0,0 +1,178 @@
+
+#### High priority
+
+* Service outages of user facing services
+* Slowdown or degradation of service on a user facing service
+* Failure of a system that stops one or more teams from working
+* Failure of a system that creates a major vulnerability in the cloud platform
+
+#### Postmortems
+
+In the cloud platform team we follow the practice of better understanding system failures through [blameless postmortems](https://codeascraft.com/2012/05/22/blameless-postmortems/).
+
+The point of a postmortem is to understand the various factors that led up to a system failure and to try to identify things that can be improved to try to stop a similar event occurring in future.
+
+Our approach to postmortems is to:
+
+* For high priority incidents to hold a group session as soon as possible after the incident is closed
+* Develop a timeline of the activities that led to the incident and its resolution
+* Walkthrough that timeline and use it to identify opportunities for improvements
+* Emerge with a prioritised list of improvements to be made, with owners
+* Write up and publish the postmortem, giving a record of the timeline and any actions that have been agreed
+
+We currently publish our postmortem reports on [pagerduty](https://moj-digital-tools.pagerduty.com/postmortems/). We have agreement to make these public and are in the process of working out how to do that.
+
+#### Our on call process
+
+Team members who are on call manage an on call rota in pagerduty. The on call rota consists of a primary engineer and a secondary engineer.
+
+Team members that are new to the rota should ensure that they have a secondary who has enough knowledge and experience of the process to support them.
+
+The hours of on call are 7AM–10AM and 5PM–10PM weekdays, 7AM-10PM weekends and bank holidays. 9AM - 10AM and 5PM - 6PM are additionally covered by a team member from support who is online during those periods.
+
+On call team members will respond to high priority incidents out of hours and will work on those incidents for up to 1 hour. If the engineer is not able to resolve the issue within that timeframe they will:
+
+* put the service into maintenance mode
+* inform the affected team
+* put together a plan to resolve the issue in hours
+
+#### Our documentation
+
+For our current systems based around the Template Deploy tooling, our documentation lives in [Confluence](https://dsdmoj.atlassian.net/wiki/spaces/PLAT/overview). This documentation includes architecture, runbooks and common issues.
+
+During the development of the Cloud Platform we are putting our documentation into Git repos stored on Github. The starting point for this documentation is the [Cloud Platform repo](https://github.com/ministryofjustice/cloud-platform)
+
+#### Measuring how we are doing
```

Comment:
Please remove this section as we are not doing this. We will also be introducing OKRs with would replace these and possibly be published elsewhere.

Reply:
removed

Reply:
I think you may have removed the wrong section. We should keep "Our documentation" and remove "Measuring how we are doing" 

In our documentation we should switch the order so that the Cloud Platform docs are referenced first followed by Template Deploy.